smile_id_core/WebApi.py

```diff
@@ -116,7 +116,7 @@ def submit_job(
                 partner_params=partner_params,
                 id_info_params=id_info_params,
                 upload_url=upload_url,
-                sec_params=sec_params,
+                signature_params=sec_params,
```

Comment:
```suggestion
                signature_params=signature_params,
```

I'd imagine the declaration of `sec_params` also needs to be updated